### ISSUE TITLE
Export refuses to start with "Unable to initialize Video driver"

### ISSUE THREAD

**Issue by lekoder** [godotengine/godot#23406]
**Godot version:**
3.1 0dbe01483

**OS/device including version:**
Windows 7, GeForce 660 TI, drivers version 416.34 (or 25.21.14.1634 from 10.10.2018)

**Issue description:**
Issue reported by player [via Steam](https://steamcommunity.com/app/846030/discussions/0/1733216261957414024/). Release export refuses to start displaying:

> Unable to initialize Video driver
> Your video card driver does not support any of the supported OpenGL versions. Please update your drivers or if you have a very old or integrated GPU upgrade it.

Console output (with `-v`) shows (this is the whole output):
```
ERROR: RasterizerGLES3::is_viable: Error initializing GLAD
At: drivers\gles3\rasterizer_gles3.cpp:144
```
I have no access to the machine or any similar. Asked a volunteer to test it on similar config (same graphics card) and it worked fine. Game has "Driver Fallback: Never" setting to prevent crashes on unsupported cards (suggested by @akien-mga) - this might be related. Nevertheless, 660TI seems to meet the requirements.

**Steps to reproduce:**
Can't reproduce on any machine I have access to.

It is affecting a player of Steam-published demo. Demo itself can be downloaded [via Steam](https://store.steampowered.com/app/846030/V_Rings_of_Saturn/), if it's any help.

Please advise.

**Comment by fire** [comment]
Is the person on mobile graphics with a dedicated gpu? In that case he may have to force the gpu.

I don't know.

**Comment by lekoder** [comment]
@fire it seems to be a desktop PC. It does have an integrated card, but with desktop PCs these usually have separate video connector.

**Comment by akien-mga** [comment]
Most likely they don't have GPU drivers installed / broken ones without OpenGL 3.3 support.
You can ask them to check with e.g. GLview: http://realtech-vr.com/admin/glview

**Comment by lekoder** [comment]
@akien-mga these seem to be in order:
![overview](https://i.imgur.com/PbWQOf2.jpg)
![features](https://i.imgur.com/Gky6POm.jpg)

**Comment by akien-mga** [comment]
@hpvb Any idea what could cause the renderer to fail init even though the drivers seem OK?

**Comment by starry-abyss** [comment]
I used both Godot 3.0 and recently some 3.1 version on Windows 7 with GTX 660 (no TI suffix though), had no problems with OpenGL

**Comment by lekoder** [comment]
@starry-abyss That's most certainly not an issue with hardware, I had the game tested on same config and it worked fine. Either there is something wrong with players software setting, or there is some kind of edge case.

**Comment by hpvb** [comment]
This suggests that GLES3 fails to load, driver fallback never means it won't try with GLES2 later.

**Comment by hpvb** [comment]
Maybe it's the same issue as fixed in #23461 though?

**Comment by akien-mga** [comment]
Indeed sounds like it might have been due to #23461, so Godot was running on the integrated GPU instead of the discrete Nvidia one.

The user in the Steam thread also mentions that it's now working fine.
